src/test.js

```diff
@@ -0,0 +1,58 @@
+import './index.css';
+import Card from './components/userContent/cardTemplate/card';
+
+let props = {
+  header: {
+    weekday: 'Понедельник',
+    date: '19.04',
+    active: true,
+  },
```

Comment:
Weekday and date attributes are redundant. It's better to use a single date attribute of type Date and then calculate weekday and do Date formatting inside the component.

Reply:
To make sure the integration with the server goes smoothly use sample data which looks as close to the actual server output as possible.